src/components/Shop/Shop.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+const Shop = () => {
+  return (
+    <div style={{ backgroundImage: "url(./shopimage.png)" }}>
+      <h1 className="text-5xl text-olivegreen font-regular ml-64 mt-8">
+        Save The World!
+      </h1>
+
+      <ul className="flex justify-end">
+        <div className="flex flex-col mr-16">
+          <li className="text-3xl font-regular">
+            <span className="text-red">R</span>euse
+          </li>
+          <li className="text-3xl font-regular">
+            <span className="text-grey">R</span>educe
+          </li>
+          <li className="text-3xl font-regular">
+            <span className="text-olivegreen">R</span>ecycle
+          </li>
+        </div>
+      </ul>
+      <div className="ml-60">
```

Comment:
Avoid using margins to center items, instead use text-center or center it with justify-content if you're using flexboxes:
- https://tailwindcss.com/docs/justify-content
- https://tailwindcss.com/docs/text-align